modules/virt-configuring-vm-real-time.adoc

```diff
@@ -67,38 +67,63 @@ spec:
 $ oc apply -f <file_name>.yaml
 ----
 
-. Configure the guest operating system. The following example shows the configuration steps for a {op-system-base} 8 operating system:
+. Configure the guest operating system. The following example shows the configuration steps for a {op-system-base} 9 operating system:
 .. Run the following command to connect to the VM console:
 +
 [source,terminal]
 ----
 $ virtctl console <vm_name>
 ----
 
-.. Configure huge pages by using the GRUB boot loader command-line interface. In the following example, 8 1G huge pages are specified.
+.. Make sure irqbalance is disabled:
 +
 [source,terminal]
 ----
-$ grubby --update-kernel=ALL --args="default_hugepagesz=1GB hugepagesz=1G hugepages=8"
+$ systemctl disable irqbalance && systemctl stop irqbalance
 ----
 
-.. To achieve low-latency tuning by using the `cpu-partitioning` profile in the TuneD application, run the following commands:
+.. Install the following packages:
 +
 [source,terminal]
 ----
-$ dnf install -y tuned-profiles-cpu-partitioning
+$ dnf install kernel-rt tuned cloud-init rt-tests
+----
+
+.. To achieve low-latency tuning by using the `realtime-virtual-guest` profile in the TuneD application, run the following commands:
++
+[source,terminal]
+----
+$ dnf --enablerepo=rt install -y tuned-profiles-realtime
+$ dnf --enablerepo=nfv install -y tuned-profiles-nfv-guest
 ----
 +
 [source,terminal]
 ----
-$ echo isolated_cores=2-9 > /etc/tuned/cpu-partitioning-variables.conf
+$ echo isolated_cores=2-3 > /etc/tuned/realtime-virtual-guest-variables.conf
+$ echo isolate_managed_irq=Y >> /etc/tuned/realtime-virtual-guest-variables.conf
```

Comment:
Usually this setting has already been in the conf file, maybe it's better to have a check before setting it.